misaligned.c

```diff
@@ -1,21 +1,25 @@
-#include <stdio.h>
-#include <assert.h>
+#include "misaligned.h"
 
 int printColorMap() {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+    int majorColorIndex = 0, minorColorIndex = 0;
+    for(majorColorIndex = 0; majorColorIndex < numberOfMajorMinorColors; majorColorIndex++) {
+        for(minorColorIndex = 0; minorColorIndex < numberOfMajorMinorColors; minorColorIndex++) {
+            printf("%d | %s | %s\n", (majorColorIndex * numberOfMajorMinorColors + minorColorIndex), majorColor[majorColorIndex], minorColor[minorColorIndex]);
         }
     }
-    return i * j;
+    return  majorColorIndex * minorColorIndex;
 }
-
-int main() {
+void testprintColorMap()
+{
     int result = printColorMap();
-    assert(result == 25);
+    assert(result == COLORMAPVALUE1);

```

Comment:
As of now there is only one value that test case should be expecting i.e. a value which is product of num of major colors and number of minor colors.

Expecting 3 values for the same result is not right.